### PR TITLE
Update usage of "Swarm" for consistency

### PR DESCRIPTION
**Pull request by thaJeztah**
relates to https://github.com/docker/docker/issues/24905

ping @sfsmithcha

### CODE REVIEW

File: daemon/cluster/cluster.go [hunk 1]
```diff
@@ -519,24 +519,24 @@ func (c *Cluster) Leave(force bool) error {
...
 			if err == nil {
 				if active && reachable-2 <= unreachable {
 					if reachable == 1 && unreachable == 0 {
-						msg += "Removing the last manager will erase all current state of the cluster. Use `--force` to ignore this message. "
+						msg += "Removing the last manager erases all current state of the swarm. Use `--force` to ignore this message. "
 						c.Unlock()
 						return fmt.Errorf(msg)
 					}
-					msg += fmt.Sprintf("Leaving the cluster will leave you with %v managers out of %v. This means Raft quorum will be lost and your cluster will become inaccessible. ", reachable-1, reachable+unreachable)
+					msg += fmt.Sprintf("Leaving the swarm will leave you with %v managers out of %v. This means Raft quorum will be lost and your swarm becomes inaccessible. ", reachable-1, reachable+unreachable)
```
Comment (sfsmithcha) [inline daemon/cluster/cluster.go]: suggest:
Removing this node leaves %v managers out of %v. Without a Raft quorum your cluster will be inaccessible.

File: docs/reference/api/docker_remote_api_v1.24.md [hunk 2]
```diff
@@ -3323,7 +3323,7 @@ Instruct the driver to remove the network (`id`).
 
 ## 3.6 Nodes
 
-**Note**: Nodes operations require to first be part of a Swarm.
+**Note**: Nodes operations require to first be part of a swarm.
```
Comment (sfsmithcha) [inline docs/reference/api/docker_remote_api_v1.24.md]: Node operations

File: docs/reference/api/docker_remote_api_v1.24.md [hunk 3]
```diff
@@ -3627,9 +3627,9 @@ JSON Parameters:
   number, like `eth0:4567`. If the port number is omitted, the port number from the listen
   address is used. If `AdvertiseAddr` is not specified, it will be automatically detected when
   possible.
-- **ForceNewCluster** – Force creating a new Swarm even if already part of one.
-- **Spec** – Configuration settings of the new Swarm.
-    - **Orchestration** – Configuration settings for the orchestration aspects of the Swarm.
+- **ForceNewCluster** – Force creating a new swarm even if already part of one.
```
Comment (sfsmithcha) [inline docs/reference/api/docker_remote_api_v1.24.md]: Force creation of a new swarm?

File: daemon/cluster/cluster.go (hunk 1 above)
Comment (thaJeztah) [inline daemon/cluster/cluster.go]: oh! excellent 👍

File: docs/reference/api/docker_remote_api_v1.24.md [hunk 4]
```diff
@@ -3627,9 +3627,9 @@ JSON Parameters:
...
+- **Spec** – Configuration settings of the new swarm.
```
Comment (sfsmithcha) [inline docs/reference/api/docker_remote_api_v1.24.md]: settings _for_

File: docs/reference/api/docker_remote_api_v1.24.md [hunk 5]
```diff
@@ -3682,7 +3682,7 @@ Join an existing new Swarm
 
 - **200** – no error
 - **400** – bad parameter
-- **406** – node is already part of a Swarm
+- **406** – node is already part of a swarm
```
Comment (sfsmithcha) [inline docs/reference/api/docker_remote_api_v1.24.md]: a member of a swarm? (i think we talk about swarm membership)

File: docs/reference/api/docker_remote_api_v1.24.md [hunk 6]
```diff
@@ -3693,17 +3693,17 @@ JSON Parameters:
   number, like `eth0:4567`. If the port number is omitted, the port number from the listen
   address is used. If `AdvertiseAddr` is not specified, it will be automatically detected when
   possible.
-- **RemoteAddr** – Address of any manager node already participating in the Swarm to join.
+- **RemoteAddr** – Address of any manager node already participating in the swarm to join.
```
Comment (sfsmithcha) [inline docs/reference/api/docker_remote_api_v1.24.md]: nit: i think "to join" is awkward/unnecessary.

File: docs/reference/api/docker_remote_api_v1.24.md (hunk 5 above)
Comment (thaJeztah) [inline docs/reference/api/docker_remote_api_v1.24.md]: could work, but I think I want to make that change in a follow up; I think we use it in quite some locations

File: docs/reference/api/docker_remote_api_v1.24.md (hunk 6 above)
Comment (thaJeztah) [inline docs/reference/api/docker_remote_api_v1.24.md]: perhaps just "Address of any manager node in the swarm"?

File: docs/reference/commandline/node_promote.md [hunk 7]
```diff
@@ -19,7 +19,7 @@ Options:
       --help   Print usage
 ```
 
-Promotes a node that is pending a promotion to manager. This command targets a docker engine that is a manager in the swarm cluster.
+Promotes a node that is pending a promotion to manager. This command targets a docker engine that is a manager in the swarm.
```
Comment (sfsmithcha) [inline docs/reference/commandline/node_promote.md]: There is no more pending status cc: @aaronlehmann

File: docs/userguide/networking/get-started-overlay.md [hunk 8]
```diff
@@ -24,7 +24,7 @@ some pre-existing conditions before you can create one. These conditions are:
 Though Docker Machine and Docker Swarm are not mandatory to experience Docker
 multi-host networking, this example uses them to illustrate how they are
 integrated. You'll use Machine to create both the key-value store
-server and the host cluster. This example creates a Swarm cluster.
+server and the host cluster. This example creates a swarm.
```
Comment (sfsmithcha) [inline docs/userguide/networking/get-started-overlay.md]: not sure if we want to change all the "Docker Swarm" docs. This doesn't refer to "swarm mode"

**Comment by sfsmithcha** [comment]
except for nits LGTM

File: docs/userguide/networking/get-started-overlay.md (hunk 8 above)
Comment (thaJeztah) [inline docs/userguide/networking/get-started-overlay.md]: yeah it was in doubt, let me revert this document

**Comment by thaJeztah** [comment]
thanks @sfsmithcha; updated ptal :smile:

File: daemon/cluster/cluster.go [hunk 9]
```diff
@@ -42,16 +42,16 @@ const (
...
-var ErrSwarmExists = fmt.Errorf("This node is already part of a swarm cluster. Use \"docker swarm leave\" to leave this cluster and join another one.")
+var ErrSwarmExists = fmt.Errorf("This node is already part of a swarm. Use \"docker swarm leave\" to leave this swarm and join another one.")
 
 // ErrPendingSwarmExists is returned on initialize or join request for a cluster that is already processing a similar request but has not succeeded yet.
 var ErrPendingSwarmExists = fmt.Errorf("This node is processing an existing join request that has not succeeded yet. Use \"docker swarm leave\" to cancel the current request.")
 
 // ErrSwarmJoinTimeoutReached is returned when cluster join could not complete before timeout was reached.
-var ErrSwarmJoinTimeoutReached = fmt.Errorf("Timeout was reached before node was joined. Attempt to join the cluster will continue in the background. Use \"docker info\" command to see the current swarm status of your node.")
+var ErrSwarmJoinTimeoutReached = fmt.Errorf("Timeout was reached before node was joined. Attempt to join the swarm will continue in the background. Use \"docker info\" command to see the current swarm status of your node.")
```
Comment (aaronlehmann) [inline daemon/cluster/cluster.go]: "The attempt to join the swarm will continue in the background."

"Use the `"docker info" command to see the current swarm status of your node."

File: daemon/cluster/cluster.go [hunk 10]
```diff
@@ -519,24 +519,24 @@ func (c *Cluster) Leave(force bool) error {
...
-					msg += fmt.Sprintf("Leaving the cluster will leave you with %v managers out of %v. This means Raft quorum will be lost and your cluster will become inaccessible. ", reachable-1, reachable+unreachable)
+					msg += fmt.Sprintf("Removing this node leaves %v managers out of %v. Without a Raft quorum your swarm will be inaccessible. ", reachable-1, reachable+unreachable)
 				}
 			}
 		} else {
 			msg += "Doing so may lose the consensus of your cluster. "
 		}
 
-		msg += "The only way to restore a cluster that has lost consensus is to reinitialize it with `--force-new-cluster`. Use `--force` to ignore this message."
+		msg += "The only way to restore a swarm that has lost consensus is to reinitialize it with `--force-new-cluster`. Use `--force` to ignore this message."
```
Comment (aaronlehmann) [inline daemon/cluster/cluster.go]: "Use `--force` to suppress this message."

File: docs/reference/api/docker_remote_api_v1.24.md [hunk 11]
```diff
@@ -3323,7 +3323,7 @@ Instruct the driver to remove the network (`id`).
 
 ## 3.6 Nodes
 
-**Note**: Nodes operations require to first be part of a Swarm.
+**Note**: Node operations require to first be part of a swarm.
```
Comment (aaronlehmann) [inline docs/reference/api/docker_remote_api_v1.24.md]: "Node operations require the engine to be part of a swarm."

File: docs/reference/api/docker_remote_api_v1.25.md [hunk 12]
```diff
@@ -3324,7 +3324,7 @@ Instruct the driver to remove the network (`id`).
 
 ## 3.6 Nodes
 
-**Note**: Nodes operations require to first be part of a Swarm.
+**Note**: Node operations require to first be part of a swarm.
```
Comment (aaronlehmann) [inline docs/reference/api/docker_remote_api_v1.25.md]: "Node operations require the engine to be part of a swarm."

File: docs/reference/api/docker_remote_api_v1.24.md [hunk 13]
```diff
@@ -4299,7 +4299,7 @@ Update the service `id`.
 
 ## 3.9 Tasks
 
-**Note**: Tasks operations require to first be part of a Swarm.
+**Note**: Tasks operations require to first be part of a swarm.
```
Comment (aaronlehmann) [inline docs/reference/api/docker_remote_api_v1.24.md]: "Task operations require the engine to be part of a swarm."

File: docs/reference/api/docker_remote_api_v1.25.md [hunk 14]
```diff
@@ -4300,7 +4300,7 @@ Update the service `id`.
 
 ## 3.9 Tasks
 
-**Note**: Tasks operations require to first be part of a Swarm.
+**Note**: Tasks operations require to first be part of a swarm.
```
Comment (aaronlehmann) [inline docs/reference/api/docker_remote_api_v1.25.md]: "Task operations require the engine to be part of a swarm."

**Comment by thaJeztah** [comment]
updated and rebased; @aaronlehmann PTAL

**Comment by aaronlehmann** [comment]
LGTM

**Comment by thaJeztah** [comment]
docs CI failures can be ignored

**Comment by SvenDowideit** [comment]
@tiborvass this contains code changes - so I'm throwing it into 1.2.1 & cherry-pick - please change as needed :)

**Pull request merged**